Gruntfile.js

```diff
@@ -41,12 +40,12 @@ module.exports = function (grunt) {
 
     lintThese: [
       'Gruntfile.js',
-      '<%= root %>/tasks/**/*.js',
-      '<%= root %>/test/**/*.js',
-      '<%= src %>/**/*.js',
-      '!<%= src %>/ui/public/angular-bootstrap/**/*.js',
-      '!<%= src %>/fixtures/**/*.js',
-      '!<%= root %>/test/fixtures/scenarios/**/*.js'
+      'bin',
+      'config',
+      'src',
+      'tasks',
+      'test',
+      'utilities',
```

Comment:
This array needs to be defined with the root and a globbing pattern for `lint_staged_files.js` to pick up the staged files:
```
lintThese: [
      '<%= root %>/Gruntfile.js',
      '<%= root %>/bin/**/*.js',
      '<%= root %>/config/**/*.js',
      '<%= root %>/src/**/*.js',
      '<%= root %>/tasks/**/*.js',
      '<%= root %>/test/**/*.js',
      '<%= root %>/utilities/**/*.js',
    ],
```